website/project/views/node.py

```diff
@@ -781,6 +789,7 @@ def _view_project(node, auth, primary=False):
             },
             'alternative_citations': [citation.to_json() for citation in node.alternative_citations],
             'has_draft_registrations': node.has_active_draft_registrations,
+            'contributors': contributor_ids
```

Comment:
Notice the list comprehension just above, for `alternative_citations`. Could do the same here.